src/olympia/reviewers/views.py

```diff
@@ -650,26 +643,23 @@ def review(request, addon, channel=None):
         ),
         5,
     ).page(1)
-    if channel == amo.RELEASE_CHANNEL_LISTED:
-        if was_auto_approved:
-            try:
-                approvals_info = addon.addonapprovalscounter
-            except AddonApprovalsCounter.DoesNotExist:
-                pass
-
-        if content_review:
-            queue_type = 'content_review'
-        elif promoted_group == RECOMMENDED:
-            queue_type = 'recommended'
-        elif was_auto_approved:
-            queue_type = 'auto_approved'
-        elif is_static_theme:
-            queue_type = form.helper.handler.review_type
-        else:
-            queue_type = 'extension'
-        redirect_url = reverse('reviewers.queue_%s' % queue_type)
+    approvals_info = None
+    if (
+        channel == amo.RELEASE_CHANNEL_LISTED
+        and addon.current_version
+        and addon.current_version.was_auto_approved
+    ):
+        try:
+            approvals_info = addon.addonapprovalscounter
+        except AddonApprovalsCounter.DoesNotExist:
+            pass
```

Comment:
nit: Not a new thing, but we can move this block way down, to avoid doing this when we're redirecting - it seems to be only useful when displaying the template.

Reply:
https://github.com/mozilla/addons-server/pull/19467/commits/534a011218f67ea9af1a6a25b6ebce47b540612f